sdk/runanywhere-kotlin/docs/KOTLIN_MAVEN_CENTRAL_PUBLISHING.md

```
@@ -0,0 +1,370 @@
+}
+```
+
+### Verify Native Libraries Are Included
+
+After building, check the AAR contents:
+```bash
+unzip -l ~/.m2/repository/io/github/sanchitmonga22/runanywhere-sdk-android/VERSION/runanywhere-sdk-android-VERSION.aar | grep ".so"
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)